bin/manifest.py

```diff
@@ -0,0 +1,244 @@
+#!/usr/bin/env python
+
+import base64, os, random, sys
+from ripple import ed25519
+from ripple import ecdsa
+from hashlib import sha256
+
+#-----------------------------------------------------------
+
+# 58 character alphabet used for Ripple
+human_alphabet = 'rpshnaf39wBUDNEGHJKLM4PQRST7VWXYZ2bcdeCg65jkm8oFqi1tuvAxyz'
+
+def base_encode(b, digits, check = 1):
+    """ Returns binary as a radix-encoded string """
+    lb = b.lstrip(b'\0')
+    bn = long(0)
```

Comment:
There's no need to cast to a long, Python will automatically upcast it to you if it needs to.